liiatools/cans_pipeline/spec/CANS_schema.yml

```diff
@@ -0,0 +1,330 @@
+column_map:
+    CANS_6_21:
```

Comment:
I would suggest we change this to lowercase, cans_6_21 so stay consistent with other table names